js/feedPosts.mjs

```diff
@@ -0,0 +1,157 @@
+"use strict";
+
+const url = "https://nf-api.onrender.com";
```

Comment:
Kan ligge i en egen modul. Ikke feil det som er gjort her heller.

Reply:
Added a module for base url. The module is however not implementet to all pages yet